### ISSUE TITLE
Are certain (categories of) members entitled to AB/TAG representation?

### ISSUE THREAD

**Issue by chaals** [w3c/w3process#495]
Some comments in #484 suggest that if e.g. Browser Makers are not represented on TAG and AB, the elections have produced the "wrong" result. That's a proposition I partially agree with: if there are no browser makers out of a dozen-odd members of an elected council in W3C, I think W3C has a problem. 

I don't think it has been a concrete problem for about a decade, and before that I think the problem was that browser makers didn't stand candidates for election. But the question raises the issue of whether there should be certain "categories" that we **ensure** are represented on AB / TAG. And this has been a topic of informal discussions in W3C many times over the last couple of decades, so I think we should raise it as a formal issue.

Apart from browser makers, over many years suggestions have included the areas we group as "horizontal review" (accessibility, internationalisation, privacy, security), various axes of diversity based on individual characteristics - gender, national/regional origin, particular industry, etc.

Obviously, there are many ways to reserve places for any category of people we can define, but they all involve two tasks:
- defining who falls into the relevant category
- choosing a mechanism to ensure they have representation (there are very many possibilities here).

Related issues include #465, #484, #486 (and historical issues #60, #190, ...)

**Comment by michaelchampion** [comment]
"Entitled" is not a useful concept here.  I'd reframe the question as "What mix of engagement, skills and perspectives should the TAG and AB selection process optimize for to ensure the *effectiveness* of those bodies in helping W3C execute its mission?"  An finely "representative" body that can't build consensus on how to solve actual problems, and persuade the larger community to adopt their solutions, will just hasten W3C's decline. 

Approaching this by trying to specify a mix of "categories of members" seems unlikely to succeed.  As @bkardell notes in https://github.com/w3c/w3process/issues/484#issuecomment-762475803
>  What is optimal for any one election inevitably involves all sorts of complex factors, from considering the current state of the TAG, standards, technology and the web.

For example, when the primary challenge to the web was the lack of interoperability across implementations of the same specs, it was imperative for W3C's governing bodies to have representatives who understood the spec and implementation issues in enough detail to find solutions.  Looking ahead we can expect other challenges to be more important to W3C, especially finding a workable balance between privacy concerns and user information to support the advertising business model that supports most "free" websites.  The right mix of skills, diversity of perspectives, and willingness to commit time to W3C changes with every election, and the criteria for selecting people needs to be flexible.

An optimal mix of skills and perspectives for the new challenges will probably require different *people* .... but not necessarily different *members*:  the same companies that make browsers tend to be those with business models that find a sweet spot among competing concerns.  (For example Apple AFAIK does not have an ad business, but successfully persuades customers to give them lots of money partly by credibly promising to respect their privacy).

Finally, I really have to push back on the idea that "browser makers" are some sort of faction that collectively only needs one or two representatives on the TAG and AB. Sure, Mozilla is a 'browser maker", and Google,  Apple, Microsoft, "make browsers", but the latter invest only a small percentage of their resources in browser making.  And I know from personal experience that the management at least one of those companies sees the others as their most feared competitors, not as allies.

What  really distinguishes the so-called "browser makers" from most members is their *willingness to invest resources in W3C*.  Look at https://www.w3.org/organizations/ -- the only members with more than 50 representatives engaged in W3C are the "browser makers" (plus Intel, which AFAIK contributes heavily to browser implementation in the open source projects). I think you'll see the same pattern in TPAC membership for the last several years.

So, I vigorously disagree that W3C would help its mission by making it difficult for the members who invest the most human capital in W3C to win seats on the governance bodies.

**Comment by dwsinger** [comment]
Like Mike, I'd rather ask the question the other way up. How do we achieve balance, and avoid a [shoe event horizon](https://hitchhikers.fandom.com/wiki/Shoe_Event_Horizon), i.e. all the participants on the group are similar, and needed skills or perspectives are missing?

Indeed, if browser vendors are missing, we should be concerned; but that's not the only group.

**Comment by fantasai** [comment]
Glad to have this issue open for discussion as it indeed has come up multiple times, but I think my position is that quotas are something to be avoided unless we have a real problem we can't solve by other means. There are many axes of representation that we want the AB/TAG/etc. to embody, and I don't think it's good for us to pin down a few of them and encode them into the election system. If we have a recurring problem like multiple elections and no browser vendor gets elected but we all feel that at least one ought to, or multiple elections and no candidate from category Y gets elected but we all feel that at least one ought to, then it's time for quotas. But until then, I think it's up to us to balance the system as best we can taking each election as it comes, and doing our best to recruit and promote good people. Quotas are a blunt instrument, and I'd rather avoid them if we can.

**Comment by frivoal** [comment]
Agree with @fantasai.

Besides that, I don't think browser vendors for a uniform group. Mozilla, Apple, and Google have hugely different product offerings and business strategies. I don't think it makes a whole lot of sense to treat them as a single constituency. They do all ship a similar product (a browser), and because of that implement a great many W3C specs, but they are very different from each other.

With that said, I do think it would be noteworthy if our election results consistently failed to elect representatives from these companies. I don't mind if some of them lose some of the time, that's to be expected with elections. But they are among the most active participants, whether you measure it by spec editors, test case writers, implementation developers, person active in w3c, or a whole host of other metrics. For sure, the w3c community is home to much more than just browser vendors, but they nonetheless are central figures to what we do, and if all / most of them were repeatedly losing elections, it would be worth wondering why.

**Comment by dwsinger** [comment]
Agreed, we should not have quotas, or a sense of entitlement. Yes, the browser vendors have different perspectives and positions, but there are always questions/statements where you might hear one of them saying "I don't see that getting traction in browsers" (i.e. an ability to speak generally, from their knowledge of the field) — and there are other constituencies that have felt able to make similar general statements ("I think it unlikely that this would get adopted in the authoring tools", "I don't see web sites revising to come into line with this even if it were mandated", and so on).

**Comment by dwsinger** [comment]
The CG is asking whether there is a concrete proposal here; absent such, we expect to close soon
